cloud/openstack/os_server.py

```diff
@@ -555,6 +560,7 @@ def main():
         boot_volume                     = dict(default=None, aliases=['root_volume']),
         terminate_volume                = dict(default=False, type='bool'),
         volumes                         = dict(default=[], type='list'),
+        scheduler_hints                 = dict(default=None),
```

Comment:
This is supposed to be a dict, yeah? We need to be explicit about the type here. Otherwise a 'str' is assumed.